examples/ingest/cmd/single.go

```diff
@@ -38,19 +39,16 @@ var singleCmd = &cobra.Command{
 			Operation: &ingestpb.Record_Upsert{
 				Upsert: &ingestpb.UpsertData{
 					Data: &ingestpb.UpsertData_Node{
-						Node: &ingestpb.Node{
-							Type: &ingestpb.Node_Resource{
-								Resource: &ingestpb.Resource{
-									ExternalId: "0001",
-									Type:       "Truck",
-									Properties: []*ingestpb.Property{
-										{
-											Key: "vin",
-											Value: &objects.Value{
-												Value: &objects.Value_IntegerValue{
-													IntegerValue: 1234,
-												},
-											},
+						Node: &knowledgeobjects.Node{
+							ExternalId: "0001",
+							Type:       "Truck",
+							IsIdentity: false,
```

Comment:
nitpick: if you omit boolean values in Go, they default to false. So no need to explicitly set "false"